src/redisearch_rs/redis_mock/build.rs

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+fn main() {
+    // link `librust_binded.a` to this crate `redis_mock`
+
+    // get variables to determine paths
+    let os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
+    let arch = std::env::var("CARGO_CFG_TARGET_ARCH")
+        .unwrap()
+        .replace("x86_64", "x64");
+    let platform = format!("{}-{}-{}", os, arch, get_build_profile_name());
+
+    let root = git_root();
+    let lib_path = root
```

Comment:
if we depend on the rust binding artifact in the build script we need to add it as a dependency to Cargo.toml too.